dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java

```diff
@@ -179,20 +187,31 @@ public void testNotKnownEndpoint() throws Exception {
                         .expectCode(200)
                         .run(String.class);
 
-        assertFalse(server.waitForStringInLogUsingMark("Failed to export spans. The request could not be executed. Full error message:.*" + INVALID_JAEGER_ENDPOINT.toLowerCase())
-                        .isEmpty());
+        if(RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP61_ID)){
+            assertFalse(server.waitForStringInLogUsingMark("Failed to export spans. Server responded with gRPC status code 2. Error message: " + INVALID_JAEGER_ENDPOINT.toLowerCase())
+            .isEmpty());
+        }
+        else{
+            assertFalse(server.waitForStringInLogUsingMark("Failed to export spans. The request could not be executed. Full error message:.*" + INVALID_JAEGER_ENDPOINT.toLowerCase())
+            .isEmpty());
+        }
```

Comment:
`waitForStringInLogUsingMark` returns `null` if the line is not found which would result in an NPE with the current code.

```suggestion
        if(RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP61_ID)){
            assertNotNull(server.waitForStringInLogUsingMark("Failed to export spans. Server responded with gRPC status code 2. Error message: " + INVALID_JAEGER_ENDPOINT.toLowerCase()));
        }
        else{
            assertNotNull(server.waitForStringInLogUsingMark("Failed to export spans. The request could not be executed. Full error message:.*" + INVALID_JAEGER_ENDPOINT.toLowerCase()));
        }
```